pkg/pillar/cmd/nodeagent/handletimers.go

```diff
@@ -322,5 +322,13 @@ func handleNodeReboot(ctxPtr *nodeagentContext) {
 	log.Functionf("Timer Expired.. Zboot.Reset()")
 	syscall.Sync()
 	<-rebootTimer.C
+	rebootTimer = time.NewTimer(duration)
+	go func() {
+		// in case of problems inside zboot.Reset
+		// we will stop zedbox process after delay
+		// and wait for watchdog to fire
+		<-rebootTimer.C
+		os.Exit(0)
```

Comment:
Please add a log.Errorf here so we can tell this happens.

Reply:
Added